pkg/reconciler/knativeserving/common/gateway.go

```diff
@@ -23,9 +24,19 @@ func GatewayTransform(instance *servingv1alpha1.KnativeServing, log *zap.Sugared
 }
 
 func updateKnativeIngressGateway(gatewayOverrides servingv1alpha1.IstioGatewayOverride, u *unstructured.Unstructured) error {
-	if len(gatewayOverrides.Selector) > 0 {
+	if len(gatewayOverrides.Selector) > 0 || len(gatewayOverrides.Servers) > 0 {
```

Comment:
First, I hope the logic of gatewayOverrides.Selector and gatewayOverrides.Servers can be separated:
if len(gatewayOverrides.Selector) > 0, we do ...
if len(gatewayOverrides.Servers) > 0, we do ...

Reply:
Thank you, I updated the code